api/api/utils/watermark.py

```diff
@@ -190,8 +190,13 @@ def _print_attribution_on_image(img: Image.Image, image_info):
 
     font = ImageFont.truetype(_get_font_path(), size=font_size)
 
+    lic = image_info["license"]
```

Comment:
Maybe add a "normalize license" function somewhere in utils that would convert these to the expected values? Then it will be easier to remove it once the catalog data has been fixed.

I would also prefer a try/catch around the `License` calls for now to ensure that we don't return 500 if there are any other problems with the production data.

Reply:
We could probably add a very permissive implementation of something like `get_License` function for now, watch the logs for errors, and open an issue to make the function stricter once we know that the data is normalized.
This would allow us to deploy the API with this fix without having to worry about rollback.